Sources/DocCDocumentation/DocCDocumentation.docc/V5_API_Migration_guide.md

```diff
@@ -83,3 +83,7 @@ The minimum targets have been raised to the folllowing:
 ## Release Assets
 
 Pre-built `.frameworks` are no longer included in releases, only `.xcframeworks`: https://github.com/RevenueCat/purchases-ios/pull/3582
+
+## Breaking Changes
+
+- The scope of `onChangeOf` is changed from `public` to `internal`
```

Comment:
🤔 This would be something like: 
```suggestion
- The scope of the View extension`onChangeOf(value:perform:)` is changed from `public` to `internal`
```
right? 

Also, maybe we can also add a note stating that you most likely wouldn't have used it? Just wondering if we can save folks a few minutes of spelunking their codebase for usage of a thing they probably never touched